test/activations.jl

```diff
@@ -332,39 +320,57 @@ end
         gradtest(f, +2 .+ rand(rng, 2, 2), check_broadcast=true)
         gradtest(f, -2 .- rand(rng, 2, 2), check_broadcast=true)
 
-        if Symbol(f) in NNlib.UNARY_ACTS
+        if f in UNARY_FUNCTIONS
             @test rrule(f, rand()) !== nothing
             @test rrule(broadcasted, f, rand(2)) !== nothing
         end
-        if Symbol(f) in NNlib.BINARY_ACTS
-            @test rrule(f, rand(), rand()) !== nothing
-            @test rrule(broadcasted, f, rand(2), rand(2)) !== nothing
+        if f in BINARY_FUNCTIONS
+            @testset "binary rule" begin
+                ## Check that rules, including broadcast rules, are defined:
+                @test rrule(f, rand(), rand()) !== nothing
+                @test rrule(broadcasted, f, rand(2), rand(2)) !== nothing
+
+                ## Correctness tests above don't check 2-arg version.
+                gradtest(x -> f(x, 0.2), 1 + rand(rng))
+                gradtest(x -> f(x, 0.7), 1 + rand(rng))
```

Comment:
Old binary tests didn't attempt to check correctness, only that t rule existed. These ones do.

There is also a test of the gradient rule below.